src/cryptominisat.cpp

```diff
@@ -494,32 +494,34 @@ static bool actually_add_clauses_to_threads(CMSatPrivateData* data)
 
 DLL_PUBLIC void SATSolver::set_max_time(double max_time)
 {
-  for (size_t i = 0; i < data->solvers.size(); ++i) {
-    Solver& s = *data->solvers[i];
-    if (max_time >= 0) {
-      // the main loop in solver.cpp is checks `maxTime`
-      // against `cpuTime`, so we specify `s.conf.maxTime`
-      // as an offset from `cpuTime`.
-      s.conf.maxTime = cpuTime() + max_time;
-
-      //don't allow for overflow
-      if (s.conf.maxTime < max_time)
-          s.conf.maxTime = max_time;
-    }
+  if (max_time < 0) {
+    return;
+  }
```

Comment:
Please assert fail. It should be not allowed. Silent failure is a bad idea, people will be chasing their tails. Thanks!